sql/base/auth_database.sql

```diff
@@ -3291,10 +3293,6 @@ SET @saved_cs_client     = @@character_set_client;
  1 AS `Permission`*/;
 SET character_set_client = @saved_cs_client;
 
---
```

Comment:
Be more careful when staging changes in git, this shouldn't be in the diff (as well as timestamp at end of file)

Reply:
You got it.